src/fivetran-destination/src/destination/dml.rs

```diff
@@ -388,6 +407,152 @@ async fn update_file(
     Ok(())
 }
 
+async fn delete_files(
+    schema: &str,
+    table: &Table,
+    file_config: &FileConfig,
+    client: &tokio_postgres::Client,
+    delete_files: &[String],
+) -> Result<(), anyhow::Error> {
+    // TODO(parkmycar): Make sure table exists.
+    // TODO(parkmycar): Retry transient errors.
+
```

Comment:
Can this have a quick return for `if delete_files.is_empty()` to avoid any work at all, especially the CREATE TABLE?

Reply:
Added!